src/state/slices/portfolioSlice/portfolioSlice.ts

```diff
@@ -174,32 +172,47 @@ export const portfolioApi = createApi({
   // refetch if network connection is dropped, useful for mobile
   refetchOnReconnect: true,
   endpoints: build => ({
-    getAccount: build.query<Portfolio, GetAccountArgs>({
-      queryFn: async ({ accountSpecifierMap }, { dispatch, getState }) => {
-        if (isEmpty(accountSpecifierMap)) return { data: cloneDeep(initialState) }
+    getAccounts: build.query<Portfolio, GetAccountsArgs>({
```

Comment:
please don't change this from a single call to multiple, we want to maintain the granularity here